### PR TITLE
WIP [generate:content] New command

### PR DESCRIPTION
**Pull request by enzolutions**

### CODE REVIEW

**Comment by darol100** [comment]
@enzolutions , @jmolivas This seem to be specific to nodes is that right ? If so... do you think it will be better to name this command to **generate:content:node** so in the future we can create other types of content like taxonomy ?

File: src/Command/Generate/ContentCommand.php [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+        /*print 'Content types:';
+        print_r($contentTypes);
+        print '\n';*/
+
+        $this->getContentTypes();
+
+        //print_r($this->contentTypes);
+
+        if ($contenTypes = array_intersect(array_keys($this->contentTypes), $contentTypes)) {
```
Comment (miromichalicka) [inline src/Command/Generate/ContentCommand.php]: typo?

**Comment by jmolivas** [comment]
I am fixing this command, sending a new PR soon.

**Comment by jmolivas** [comment]
The `generate:content` command is almost ready:

drupal-generate-content

**Comment by jmolivas** [comment]
Command name changed to `create:nodes`
drupal-create-nodes

**Pull request merged**